source/class/cv/ui/structure/pure/NavBar.js

```diff
@@ -56,8 +56,9 @@ qx.Class.define('cv.ui.structure.pure.NavBar', {
       init: 'left'
     },
     dynamic: {
-      check: "Boolean", 
-      init: false
+      check: [true, false, null],
```

Comment:
This change is not needed, the "Boolean" check allows `null` values when `nullable` is `true`.